app/root.tsx

```diff
@@ -10,16 +17,32 @@ export { headers, meta } from './__root.client.tsx'
 export { action, loader } from './__root.server.tsx'
 
 export default function App() {
+	const data = useLoaderData<typeof loader>();
 	const nonce = useNonce()
-
+	const theme = useTheme()
 	return (
-		<Document nonce={nonce}>
+
+		<Document nonce={nonce} theme={theme}>
+
 			<div className="flex h-screen flex-col justify-between">
+			<HeaderWithSearch />
 				<div className="flex-1">
 					<main className="grid h-full place-items-center">
-						<h1 className="text-mega">Welcome to Epic News!</h1>
+						<h1 className= "text-mega text-pink-500">Your Journey Begins!</h1>
+						<p className='text-base text-pink-500 md:text-lg lg:text-xl'>
+							Welcome to Epic News, where the latest developments in tech are found.</p>
+							<button className="bg-indigo-500 ... px-8 py-4 hover:bg-blue-400">
+ 								 Learn More
+						</button>
 					</main>
 				</div>
+				
+				<div className="container flex justify-between pb-5">
+          <ThemeSwitch userPreference={data.requestInfo.userPrefs.theme} />
+        </div>
+		
+       		 <FooterMenuRight />
```

Comment:
Can you add your logo to the footer as well? See the [Code Monster guide](https://gedsargent.github.io/code-monster/guides/epic-news/ui/import-image-to-navbar-and-footer/#importing-the-image) for help with this.